client/index.html

```diff
@@ -86,7 +86,5 @@ <h1 class="mt-3 mb-5" data-translate="preload_manager_ovh"></h1>
                 </div>
             </div>
         </div>
-
-        <div data-otrs-popup></div>
```

Comment:
This change should have its own commit

Reply:
The OTRS module was previously glued with `@ovh-ux/manager-core` which is no more the case.

Since @marie-j has upgraded the `@ovh-ux/manager-navbar` (which have `@ovh-ux/manager-core` has a peerDependency) the `otrs-popup` directive can be removed safely.